lib-multisrc/comicaso/src/eu/kanade/tachiyomi/multisrc/comicaso/Comicaso.kt

```diff
@@ -0,0 +1,502 @@
+            "Kids" -> "kids"
+            "Magic" -> "magic"
+            "Magical Girls" -> "magical-girls"
+            "Manga" -> "manga"
+            "Manhua" -> "manhua"
+            "Manhwa" -> "manhwa"
+            "Martial Arts" -> "martial-arts"
+            "Mature" -> "mature"
+            "Medical" -> "medical"
+            "Military" -> "military"
+            "Monster Girls" -> "monster-girls"
+            "Monsters" -> "monsters"
+            "Music" -> "music"
+            "Mystery" -> "mystery"
+            "NTR", "Ntr" -> "ntr"
+            "Non-human", "Non Human" -> "non-human"
+            "Office Workers" -> "office-workers"
+            "Omegaverse" -> "omegaverse"
+            "Oneshot" -> "oneshot"
+            "Philosophical" -> "philosophical"
+            "Police" -> "police"
+            "Psychological" -> "psychological"
+            "Regression" -> "regression"
+            "Reincarnation" -> "reincarnation"
+            "Revenge" -> "revenge"
+            "Reverse Harem" -> "reverse-harem"
+            "Reverse Isekai" -> "reverse-isekai"
+            "Romance" -> "romance"
+            "Royal family", "Royal Family" -> "royal-family"
+            "Royalty" -> "royalty"
+            "School Life" -> "school-life"
+            "Sci-Fi", "Sci Fi" -> "sci-fi"
+            "Seinen(M)", "Seinenm" -> "seinenm"
+            "Sejarah" -> "sejarah"
+            "Shoujo(G)", "Shoujog" -> "shoujog"
+            "Shoujo ai", "Shoujo Ai" -> "shoujo-ai"
+            "Shounen ai", "Shounen Ai" -> "shounen-ai"
+            "Shounen(B)", "Shounenb" -> "shounenb"
+            "Showbiz" -> "showbiz"
+            "Slice of Life", "Slice Of Life" -> "slice-of-life"
+            "SM/BDSM/SUB-DOM", "SM Bdsm Sub Dom" -> "sm-bdsm-sub-dom"
+            "Smut" -> "smut"
+            "Space" -> "space"
+            "Sports" -> "sports"
+            "Super Power" -> "super-power"
+            "Superhero" -> "superhero"
+            "Supernatural" -> "supernatural"
+            "Survival" -> "survival"
+            "Thriller" -> "thriller"
+            "Time Travel" -> "time-travel"
+            "Tower Climbing" -> "tower-climbing"
+            "Tragedy" -> "tragedy"
+            "Transmigration" -> "transmigration"
+            "Vampires" -> "vampires"
+            "Video Games" -> "video-games"
+            "Villainess" -> "villainess"
+            "Violence" -> "violence"
+            "Western" -> "western"
+            "Wuxia" -> "wuxia"
+            "Yakuzas" -> "yakuzas"
+            "Yaoi(BL)", "Yaoibl" -> "yaoibl"
+            "Yuri(GL)", "Yurigl" -> "yurigl"
+            "Zombies" -> "zombies"
+            else -> genre.lowercase()
+        }
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val json = JSONObject(response.body.string())
+        val items = json.getJSONArray("items")
+        return MangasPage(parseMangaList(items), items.length() == pageSize)
+    }
```

Comment:
Duplicate, you should reuse code:
```suggestion
    override fun searchMangaParse(response: Response): MangasPage =
        popularMangaParse(response)
```

(might not be correct lint-wise)